### DISCUSSION TITLE
Image Callout

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1148] [Proposals]
### Goal

Incluir nas Variants uma propriedade de imagem (icon/image) e possibilidade de desligar o título (esse opção está como possível no Brand Factory).

### Context

Dentro do fluxo de assinaturas, o componente Callout precisa ser aplicado dessa forma para a pessoa usuária. Atualmente o Callout que temos no Mística não suporta imagem e a opção de desligar (Figma).

### Impact

Proposta Work & Co - Novo App.

### Discussion Questions

Esse update poderia ser implementado no Mística? Qual equipe ficaria responsável pelo desenvolvimento.

### Do you have a visual proposal?

https://www.figma.com/file/seiNjy7YmPWEY3BfY4WYpp/Novos-componentes---Sprint-1?node-id=4%3A10046&t=gYOdlKhORV0w5rhF-1

**Comment by aweell** [comment]
Buenas @annefranca,

Resuelvo una de las dudas:

La posibilidad de ocultar el titulo es un fix que llegara en la versión [8.0.0](https://github.com/Telefonica/mistica-design/milestone/17?closed=1) y está resuelta en https://github.com/Telefonica/mistica-design/issues/997

**Reply by annefranca** [reply]
Obrigada :)

No aguardo do update da aplicação da imagem.

**Reply by aweell** [reply]
Buenas @annefranca,

A día de hoy es posible ya incluir en un callout una imagen.

Ejemplo: 

https://rebrand.ly/mdztxo9

El callout se añade a una lista de componentes que actualmente tienen una prop `icon` que se deprecará y se cambiará por la prop `asset`. en este sentido queda claro que componentes permiten diferentes assets y cuales solo admiten un icono.

En el caso de Figma es posible también cambiar el asset por una imagen a través del assetWrapper, lo que sí habría que actualizar las specs para recoger este caso, que a día de hoy no está incluido.

Creo una issue para actualizar specs.

**Reply by annefranca** [reply]
@aweell 

Essa atualização para inserir uma imagem seria responsabilidade do seu time ou do time da Vivo?
